packages/create-redwood-app/template/scripts/prerender.ts

```diff
@@ -0,0 +1,18 @@
+// import { db } from '$api/src/lib/db'
+
+import type { AvailableRoutes } from '@redwoodjs/router'
+
+// TODO: This isn't picking up the generated names in .redwood/. How can I make
+// this work?
+type RouteNames = keyof AvailableRoutes
+
+export default async function prerenderPathParameterValues(): Promise<
+  Record<RouteNames, Array<Record<string, unknown>>>
+> {
+  return {
+    // In a real app you'd probably fetch a subset of your entries in your db
+    // (using the commented db import above, or by accessing a service) and
+    // return those ids here.
+    userExample: [{ id: 1 }, { id: 2 }, { id: 3 }],
```

Comment:
On my first read, I got a little confused by what the purpose of these values were. I'm not sure how we make it a little more obvious that these values are used to feed the query variables, especially when we might use a prisma query to return the ids here. 

Just food for thought really 🤔

Reply:
I'm just pointing this out because it took a little while for me to wrap my head around this. So not doing it to try to be nitpicky 

"these values are used to feed the query variables"
It's more accurate to say
"these values are used to feed the url path parameters"
Those, in turn, are most likely passed to a cell to be used as query variables. But they don't have to be. And they can be modified before passing them on to the cell(s). That's also why I think it makes more sense to colocate with pages instead of cells.